charts/open-zaak/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: open-zaak
 description: Productiewaardige API's voor Zaakgericht Werken
 
 type: application
-version: 0.8.0
-appVersion: "1.6.0"
+version: 0.9.0
+appVersion: "1.7.0"
```

Comment:
maybe version should become `0.8.1` instead? the chart itself is not modified at all, just the `appVersion`

Reply:
I think `0.8.1` is better as well.